README.tfdoc.hcl

```diff
@@ -81,7 +81,7 @@ section {
 
       ```hcl
       module "terraform-google-artifact-registry-repository" {
-        source = "github.com/mineiros-io/terraform-google-artifact-registry-repository?ref=v0.0.3"
+        source = "github.com/mineiros-io/terraform-google-artifact-registry-repository?ref=v0.1.0"
```

Comment:
this should be postponed until release preparations that do not seem to be included in this PR